data/json/recipes/recipe_deconstruction.json

```diff
@@ -1398,6 +1449,15 @@
     "using": [ [ "soldering_standard", 20 ] ],
     "components": [ [ [ "processor", 4 ] ], [ [ "power_supply", 1 ] ], [ [ "e_scrap", 6 ] ], [ [ "solder_wire", 1 ] ] ]
   },
+  {
+    "result": "iron_pot",
+    "type": "uncraft",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "2 m",
+    "qualities": [ { "id": "SAW_M", "level": 1 } ],
+    "components": [ [ [ "sheet_metal_small", 10 ], [ "steel_chunk", 1 ] ] ]
```

Comment:
Again, based upon pure weight this should yield more. 3000 g pot & only 375g of resources. However, based upon volume this should yield less - a cumulative volume of 375ml for components with only 100ml of space for components (2100ml - 2000ml of storage space).

Reply:
Yes, 3000g is roughly 375ml of steel, then if it hold 2000ml of storage it should have a total volume of 2375ml. Then I will change the item status too.

Reply:
Yeah, you're right. I missed this the first time round.

> "//": "Density 7.60g/cm³ ~ **1.9kg/250ml** @ 50g/unit ~ stack 40 but fragments stacks poorly so only 25% of this",